files/es/web/css/syntax/index.html

```diff
@@ -27,37 +27,37 @@ <h2 id="Declaraciones_de_CSS">Declaraciones de CSS</h2>
 
 <p><img alt="css syntax - declaration.png" class="default internal" src="/@api/deki/files/6164/=css_syntax_-_declaration.png" style="border: 1px solid black; padding: 1em;"></p>
 
-<p>Hay más de <a href="/en/CSS/CSS_Reference" title="en/CSS/CSS_Reference">100 propiedades diferentes</a> en CSS y cerca de un número infinito de diferentes valores. No todos los pares de propiedades y valores son permitidos cada propiedad define que valores son válidos. Cuando un valor no es válido para una propiedad específica, la declaración es considerada <em>inválida</em> y es completamente ingorada por el motor del CSS.</p>
+<p>Hay más de <a href="/en-US/docs/Web/CSS/Reference" title="/en-US/docs/Web/CSS/Reference">100 propiedades diferentes</a> en CSS y cerca de un número infinito de diferentes valores. No todos los pares de propiedades y valores son permitidos cada propiedad define que valores son válidos. Cuando un valor no es válido para una propiedad específica, la declaración es considerada <em>inválida</em> y es completamente ingorada por el motor del CSS.</p>
```

Comment:
Las url se manejan apuntando al idioma en que estamos `es` ya sea exista o no el contenido, el bot lo redirige automáticamente en caso de no existir y cuando se cree, se corregirá automáticamente

```suggestion
<p>Hay más de <a href="/es/docs/Web/CSS/Reference" title="/es/docs/Web/CSS/Reference">100 propiedades diferentes</a> en CSS y cerca de un número infinito de diferentes valores. No todos los pares de propiedades y valores son permitidos cada propiedad define que valores son válidos. Cuando un valor no es válido para una propiedad específica, la declaración es considerada <em>inválida</em> y es completamente ignorada por el motor del CSS.</p>
```